src/main/java/com/mraof/minestuck/item/crafting/alchemy/generator/DefaultInterpreter.java

```diff
@@ -7,8 +7,12 @@
 import net.minecraft.item.ItemStack;
 import net.minecraft.item.crafting.IRecipe;
 import net.minecraft.item.crafting.Ingredient;
+import net.minecraft.item.crafting.SmithingRecipe;
+import net.minecraft.util.NonNullList;
 import net.minecraftforge.registries.ObjectHolder;
+import org.apache.logging.log4j.LogManager;
 
+import java.lang.reflect.Field;
```

Comment:
There are now unused imports from before. Revert the changes to the file/remove the imports.